### PR TITLE
feat: Use projectId and appName

### PR DESCRIPTION
**Pull request by hosekpeter**
Jira: [PSGO-499](https://keboola.atlassian.net/browse/PSGO-499)

**Changes:**
- added `projectId` and `appName` to AppConfig struct
- updated tests
- added `projectId` as an attribute to related logs

-----------

[PSGO-499]: https://keboola.atlassian.net/browse/PSGO-499?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: internal/pkg/service/appsproxy/http/router.go [hunk 1]
```diff
@@ -179,9 +179,9 @@ func (r *Router) Shutdown() {
 	r.wg.Wait()
 }
 
-func (r *Router) createConfigErrorHandler(exceptionID string) http.Handler {
+func (r *Router) createConfigErrorHandler(exceptionID string, cfg api.AppConfig) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-		r.logger.With(attribute.String("exceptionId", exceptionID)).Warn(req.Context(), `application "<proxy.appid>" has misconfigured OAuth2 provider`)
+		r.logger.With(attribute.String("exceptionId", exceptionID), attribute.String("projectId", cfg.ProjectID)).Warn(req.Context(), `application "<proxy.appid>" has misconfigured OAuth2 provider`)
```
[the remark this answers is not in the document]
Comment (michaljurecko) [reply internal/pkg/service/appsproxy/http/router.go:184]: Yes :slightly_smiling_face: but we can solve it when we have solved SameSiteStrict and error pages, or do we need to solve it ASAP?

[the remark this answers is not in the document]
Comment (enumag) [reply internal/pkg/service/appsproxy/http/router.go:184]: No, we can solve it later.

**Comment by michaljurecko** [comment]
All app telemetry attributes are now in the one place, so implementation is easy, two lines? 1. New `AppConfig` field, 2. new item in the attributes slice.

https://github.com/keboola/keboola-as-code/pull/1708#discussion_r1570968548

**Comment by michaljurecko** [comment]
@hosekpeter I included this work in https://github.com/keboola/keboola-as-code/pull/1709

So this PR can be closed.

**Pull request closed without merging**